tests/unit/dom/SrcableElement.test.js

```diff
@@ -0,0 +1,48 @@
+/* global assert, beforeEach, describe, it, TEST_URL */
+const fs = require('fs');
+const path = require('path');
+
+const helpers = require('../helpers');
+const exokit = require('../../../index');
+
+const audioData = fs.readFileSync(path.resolve(__dirname, '../data/test.ogg'), 'base64');
+const audioUri = `data:audio/ogg;base64,${audioData}`;
```

Comment:
This would be a good test, if it were used :).

I can do a follow-up with some more cases.